### ISSUE TITLE
Implement alternate EVM asset like BSC

### ISSUE THREAD

**Issue by maevyn11** [GridPlus/phonon-app#19]
In order to demonstrate phonon's capabilities effectively during testnet, it would be great to have support for a non-ETH asset that can be easily added. This way users can swap testnet ETH for another testnet asset and see that things work natively crosschain. 

I think some work was done previously to implement this asset but had been held up due to waiting on client implementation for the asset? Not sure if this was BSC or something else, any EVM compatible asset should suffice. 

Ideally this work is folded into the frontend and we can easily integrate the additional endpoint on the client to match.

**Comment by maevyn11** [comment]
Found it, looks like it was BNB and it's linked with this client issue: [](https://github.com/GridPlus/phonon-client/issues/108)
